### PR TITLE
chore: github workflow

### PR DESCRIPTION
**Pull request by pedroferreira1**

### CODE REVIEW

**Review by andreabadesso — CHANGES_REQUESTED** [review]
Maybe also add a step to code lint?

**Comment by pedroferreira1** [comment]
> Maybe also add a step to code lint?

We have no linter in the lib right now. But it's something we should add for sure in the future, then we can add to the checks

**Review by jansegre — APPROVED** [review]
Looks good, I'd recommend making a fork and pushing it to the dev branch to test a run.

**Review by msbrogli — APPROVED** [review]

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    branches:
+    - dev
+    - master
+jobs:
+  test:
+    runs-on: 'ubuntu-latest'
+    timeout-minutes: 40  # default is 360
+    strategy:
+      matrix:
+        node-version: [8.x]
```
Comment (msbrogli) [inline .github/workflows/main.yml:19]: Add `10.x`?

**Pull request merged**